sources/packages/web/src/views/aest/SearchStudents.vue

```diff
@@ -1,124 +1,155 @@
+    <content-group>
+      <v-row
+        ><v-col>
+          <v-text-field
+            label="Application Number"
+            density="compact"
+            data-cy="appNumber"
+            variant="outlined"
+            v-model="appNumber"
+            @keyup.enter="searchStudents"
+            hide-details
+          />
+        </v-col>
+        <v-col>
+          <v-text-field
+            label="SIN"
+            density="compact"
+            data-cy="sin"
+            variant="outlined"
+            v-model="sin"
+            @keyup.enter="searchStudents"
+            hide-details
+          />
+        </v-col>
+        <v-col>
+          <v-text-field
+            label="Given Names"
+            density="compact"
+            data-cy="firstName"
             variant="outlined"
-            data-cy="viewStudent"
-            @click="goToViewStudent(slotProps.data.id)"
-            >View</v-btn
+            v-model="firstName"
+            @keyup.enter="searchStudents"
+            hide-details
+          /> </v-col
+        ><v-col>
+          <v-text-field
+            label="Last Name"
+            density="compact"
+            data-cy="lastName"
+            variant="outlined"
+            v-model="lastName"
+            @keyup.enter="searchStudents"
+            hide-details
+          /> </v-col
+        ><v-col
+          ><v-btn
+            :disabled="
+              !appNumber && !firstName && !lastName && !isSINValid(sin)
+            "
+            color="primary"
+            class="p-button-raised"
+            data-cy="searchStudents"
+            @click="searchStudents()"
           >
-        </template>
-      </Column>
-    </DataTable>
+            Search
+          </v-btn></v-col
+        >
+      </v-row>
+    </content-group>
+
+    <content-group v-if="studentsFound" class="mt-8">
+      <toggle-content :toggled="!students?.length">
+        <DataTable :value="students">
+          <Column field="firstName" header="First Name" :sortable="true">
+            <template #body="slotProps">
+              <div class="p-text-capitalize">
+                {{ slotProps.data.firstName }}
+              </div>
+            </template>
+          </Column>
+          <Column field="lastName" header="Last Name" :sortable="true">
+            <template #body="slotProps">
+              <div class="p-text-capitalize">
+                {{ slotProps.data.lastName }}
+              </div>
+            </template>
+          </Column>
+          <Column field="birthDate" header="Date of Birth">
+            <template #body="slotProps">
+              <div class="p-text-capitalize">
+                {{ dateOnlyLongString(slotProps.data.birthDate) }}
+              </div>
+            </template>
+          </Column>
+          <Column header="Action">
+            <template #body="slotProps">
+              <v-btn
+                color="primary"
+                class="p-button-raised"
```

Comment:
If I am not wrong `p-button-raised` is a PrimeVue class, can we please get rid of it?

Reply:
yes, it is prime vue class